CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Changed`
 
 - [#340](https://github.com/nf-core/mag/pull/340) - Update to nf-core 2.6.1 `TEMPLATE`
+- [#350](https://github.com/nf-core/mag/pull/350) - Adds support for CheckM as alternative bin completeness and QC tool (added by @jfy133).
```

Comment:
Is there a new convention how to state individual contributions? One could also think about changing the contributions section in the `README`...

Reply:
Not exactly, but if you tag people in the README, on release they get shown as contributors to that specific release which is a nice acknowledgment of the specific work they do (rather than just the general 'catch all' on the README. I did this on the last 2 MAG releases too:

https://github.com/nf-core/mag/releases/tag/2.2.1
https://github.com/nf-core/mag/releases/tag/2.2.0

But I'm happy to remove it if you don' tlike it.

Reply:
I see, I just thought it would make sense to do this consistently

Reply:
Can also do that! Tell me which way you prefer and I'll update accordingly :)

Reply:
Did you do this in other pipelines as well? I don't mind, we can keep it like this

Reply:
Yes I'm doing it in eager, and will do in taxprofiler, funcscan etc.

Reply:
I just saw for mag 2.2.0 it was only listed for the release, while for mag 2.2.1 you tagged people for individual PRs. In any case, I think it should be done consistently, i.e. everyone would need to do it for every PR, otherwise it would be weird.

Reply:
OK, I don't mind either way. I'll suggest then mention each person for each PR in the CHANGELOG (then it makes it easier to just paste the changelog straight into the release notes and it'll already be there, rather than us having to go back and check each PR afterwards), but ultimatelythis is your pipeline ;)

Reply:
Hi, I was thinking again about this issue and if I should add my name now or not, and in the end I would somehow prefer not to start this (at least as long it's not nf-core wide like this), because then everyone who is not doing this would be excluded.

Reply:
Ah, you mean [#380](https://github.com/nf-core/mag/pull/380), sorry didn't see it.

A poll would be nice though, then we could see how to do it consistently. Even if it doesn't necessarily has to be consistent, it just easier to stick to a certain way across multiple pipelines.
So don't remove it for now.

Reply:
Oops yes, typo.

Poll here: https://nfcore.slack.com/archives/C043UU89KKQ/p1673621984306899